### PR TITLE
Adding in a smashing Dockerfile

### PR DESCRIPTION
**Pull request by bmistry12**
Making the setup more accessible by adding a Dockerfile so windows and mac users can use docker 😄

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,15 @@
+FROM python:3
+
+ARG arg_HOME=/opt/sonos-huter
+ENV HOME=$arg_HOME
+
+RUN mkdir ${HOME}
```
Comment (alexander-lloyd) [inline Dockerfile]: Should this be `mkdir -p` ?

**Review by alexander-lloyd — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 1 above)
Comment (bmistry12) [reply Dockerfile]: Yes.

Comment (bmistry12) [reply Dockerfile]: @alexander-lloyd  Is this satisfactory, or have I introduced an SQL injection?

**Comment by Jrhenderson11** [comment]
If you could add something to the README explaining how to run tool using docker that would be nice

**Pull request merged**